src/os/bluestore/BlueRocksEnv.cc

```diff
@@ -292,7 +292,7 @@ class BlueRocksWritableFile : public rocksdb::WritableFile {
   // This asks the OS to initiate flushing the cached data to disk,
   // without waiting for completion.
   // Default implementation does nothing.
-  rocksdb::Status RangeSync(off_t offset, off_t nbytes) {
+  rocksdb::Status RangeSync(uint64_t offset, uint64_t nbytes) override {
```

Comment:
The `using rocksdb::WritableFile::RangeSync;` line above needs to be removed now, correct?

Reply:
> The `using rocksdb::WritableFile::RangeSync;` line above needs to be removed now, correct?

yep, fixed. Thanks!